### ISSUE TITLE
Configuration Options?

### ISSUE THREAD

**Issue by Ryan-Berry** [Wondermall/logstash-input-gcs#1]
Would really like to use this input in a project i'm working on. Do you have a moment to include the configuration options in the readme please? 

Also, wondering if you plan to submit this / have it included in the list of inputs at Elastic: https://www.elastic.co/guide/en/logstash/current/input-plugins.html

**Comment by mmcclaskey** [comment]
Yes, a little bit of documentation would be great, thanks for sharing though.
